analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -28,6 +28,7 @@ analyzer:
       definition_file_path: "target/directories-10.pom"
       declared_licenses:
       - "Mozilla Public License 2.0"
+      declared_licenses_processed: "MPL-2.0"
```

Comment:
Commit message: Please add "Fixes #1131." to auto-close that issue upon merge.

Reply:
I will do that in the next commit which will add the same field to `Package`.